From f3f09798824985b12f7e6dfa30e28ddfc534d44c Mon Sep 17 00:00:00 2001
From: darkdarcool <darkdarcool@gmail.com>
Date: Sat, 11 May 2024 10:23:27 -0400
Subject: [PATCH] test: testing stuff

---
 src/features/hover.zig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/features/hover.zig b/src/features/hover.zig
index 109a4eb50..0f95a6a1f 100644
--- a/src/features/hover.zig
+++ b/src/features/hover.zig
@@ -124,9 +124,9 @@ fn hoverSymbolRecursive(
     const writer = hover_text.writer();
     if (markup_kind == .markdown) {
         if (is_fn) {
-            try writer.print("```zig\n{s}\n___```", .{def_str});
+            try writer.print("```zig\n{s}\n___ WOW IS_FN 127```", .{def_str});
         } else {
-            try writer.print("```zig\n{s}\n```\n```zig\n({s})\n___```", .{ def_str, resolved_type_str });
+            try writer.print("```zig\n{s}\n```\n```zig\n({s})\n___ WOW NOT_FN 129```", .{ def_str, resolved_type_str });
         }
         for (doc_strings.items) |doc|
             try writer.print("\n\n{s}", .{doc});
@@ -242,6 +242,7 @@ fn hoverDefinitionBuiltin(
                 \\{s}
                 \\```
                 \\{s}
+                \\idefk 245
             , .{ builtin.signature, builtin.documentation });
         },
     }

